frontend/src/components/Tournament.js

```diff
@@ -199,13 +199,13 @@ const Tournament = () => {
         <table class="w-full text-sm text-left text-gray-500 dark:text-gray-400">
           <tbody>
             <For each={tournamentQuery.data?.spirit_ranking}>
-              {(spirit, i) => (
+              {(spirit) => (
                 <tr class="bg-white border-b dark:bg-gray-800 dark:border-gray-700">
                   <th
                     scope="row"
                     class="pr-6 pl-10 py-4 whitespace-nowrap font-normal"
                   >
-                    {i() + 1}
+                    {spirit.rank}
```

Comment:
We need to think of having backward compatibility in this. Cause in the current completed tournaments we wont have the rank field and the rank field will only get generated if we run the `update_tournament_spirit_rankings` once.

Reply:
Correct. Currently, since there are only 2 completed tournaments, we can update any one match which would call the `update_tournament_spirit_rankings` function and resolve the rankings.

Note - for new tournaments, until the first match is updated, the ranks would not show up at all (the key `rank`) would not be present.

Reply:
Yeah cool. Also lets have a separate API to just call update_tournament_spirit_rankings just like we have to populate fixtures? We can add a button too in the Tournament Manager page to call this API. Will be helpul in the future as well when we make these kind of changes

Reply:
`/tournament/generate-fixtures/{tournament_id}`
Are you referring to this api?

Reply:
yep yep

Reply:
Don't see much usage of this additional API. Can integrate later if required as well.

Reply:
Makes sense. Let's do it later

Reply:
I've added a migration to this PR to migrate existing tournaments to use this format.